readme.md

```diff
@@ -1,18 +1,39 @@
 # Metabase Embedding React SDK nextjs sample apps
 
-This repository contains sample apps for nextjs 14 that uses Metabase embedding sdk react sdk.
-It include an app using pages router and an app using the app router.
+This repository contains sample apps for nextjs 14 that uses Metabase Embedding SDK.
+It includes an app using pages router and an app using the app router.
 
 Note: the samples are using the sdk nextjs compatibility layer, which is available from version 0.51.11.
 
 ## Instructions
 
-`cd` either `next-sample-app-router` or `next-sample-pages-router` and run `npm install`.
+### Manual run with your existing Metabase Enterprise instance
```

Comment:
I think this should say `Pro/Enterprise` as we now allow Pro licenses to be used for the SDK